packages/@glimmer/bundle-compiler/lib/bundle-compiler.ts

```diff
@@ -84,6 +85,9 @@ export interface PartialTemplateLocator<TemplateMeta> extends ModuleLocator {
   kind?: 'template';
 }
 
+// to make --declaration happy
```

Comment:
Is this a TypeScript bug?

Reply:
not sure, it felt like it. It only complained in the class initializer, when the alias was in methods it was ok